events/dvg_atlantia.txt

```diff
@@ -0,0 +1,993 @@
+				}
+				add = 50
+			}
+		}
+	}
+	option = {
+		name = dvg_atlantia.1.b
+		
+		add_loyalists = {
+			value = 0.5
+			culture = cu:lotharian
+		}
+		add_radicals = {
+			value = 0.5
+			culture = cu:amazonien
+		}
+		add_radicals = {
+			value = 0.75
+			culture = cu:atlantian
+		}
+		
+		play_as = c:LOT
+		
+		c:LOT = {
+		    
+			set_variable = tex_annexation
+			trigger_event = { id = dvg_atlantia.3 }
+			
+			show_as_tooltip = {
+				c:LOT = {
+					annex = ROOT
+				}
+			}
+		}
+		c:AMA = { trigger_event = { id = dvg_atlantia.13 popup = yes } }
+		change_tag = LOT
+
+		ai_chance = {
+			base = 25
+			modifier = {
+				trigger = {
+					"relations(scope:lot_scope)" >= relations_threshold:cordial
+				}
+				add = 10
+			}
+			
+			modifier = {
+				trigger = {
+					"relations(scope:lot_scope)" >= relations_threshold:amicable
+				}
+				add = 30
+			}				
+
+			modifier = {
+				trigger = {
+					"relations(scope:lot_scope)" >= relations_threshold:friendly
+				}
+				add = 50
+			}
+		}
+		
+	}
+	
+	option = {
+		name = dvg_atlantia.1.c
+
+		
+		c:AMA = { trigger_event = { id = dvg_atlantia.12 popup = yes } }
+		add_modifier = {
+			name = dvg_demoratic_miltias
+			days = 1825
+			is_decaying	= yes
+		}
+		add_radicals = {
+			value = 0.3
+			culture = cu:lotharian
+		}
+		add_radicals = {
+			value = 0.3
+			culture = cu:amazonien
+		}
+		add_loyalists = {
+			value = 0.55
+			culture = cu:atlantian
+		}
+
+		c:SPA = {
+			remove_diplomatic_pact = {
+				country = c:ATL
+				type = puppet
+			}
+		}
+		
+		c:ATL = {
+			change_relations = {
+				country = c:LOT
+				value = -50
+			}
+			
+			change_relations = {
+				country = c:AMA
+				value = -20
+			}
+			set_global_variable = {
+				name = atlantia_independent
+				value = 0
+			}
+		}
+		c:LOT = {
+		       
+			add_journal_entry = 
+			{ type = je_reconquering_atlantia }
+			
+		}
+		c:LOT = { trigger_event = { id = dvg_atlantia.12 popup = yes } }
+		
+
+		ai_chance = {
+			base = 65
+		}
+	}
+}
+
+#For Amazonie
+
+dvg_atlantia.2 = {
+	type = country_event
+	placement = ROOT
+	event_image = {
+		video = "gfx/event_pictures/unspecific_world_fair.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "texan_war_of_independence.1"
+
+	icon = "gfx/interface/icons/event_icons/event_portrait.dds"
+
+	title = dvg_atlantia.2.t
+	desc = dvg_atlantia.2.d
+	flavor = dvg_atlantia.2.f
+
+	duration = 3
+
+	trigger = {
+	}
+
+	immediate = {
+		c:ATL = {
+			
+			
+		}
+	}
+
+	option = {
+		name = dvg_atlantia.2.a
+		default_option = yes
+		trigger = {
+			has_variable = tex_annexation
+		}
+		
+		s:STATE_COTE_DU_CAFE = {
+			add_claim = c:LOT
+		}
+		annex = c:ATL
+		change_relations = {
+				country = c:LOT
+				value = -50
+		}
+		add_modifier = {
+			name = dvg_atlantia_support
+			days = 1825
+			
+		}
+		c:LOT = { trigger_event = { id = dvg_atlantia.4 popup = yes } }
+	}
+	ai_chance = {

```

Comment:
Error: "Unexpected token: ai_chance, near line: 301" in file: "events/dvg_atlantia.txt" near line: 303
Should be inside option.

Reply:
aight fixed